toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -297,6 +298,32 @@ func SystemdFormatCorruptionOption(corruptionOption imagecustomizerapi.Corruptio
 	}
 }
 
+func ParseSystemdVerityOptions(options string) (imagecustomizerapi.CorruptionOption, error) {
```

Comment:
nit - `parseSystemdVerityOptions(...)`, also, should we add new functions at the end of file?

Reply:
I'm placing it next to `ParseSystemdVerityOptions`, since they are they inverse of each other.

Reply:
Fixed name